package.json

```diff
@@ -99,5 +100,18 @@
     "hooks": {
       "pre-commit": "lint-staged"
     }
+  },
+  "jest": {
```

Comment:
Would be nice to set up minimum coverage with: 
```
"coverageThreshold": {
  "global": {
    "branches": 50,
     "functions": 70,
     "lines": 70,
     "statements": 70
  }
}
```

Reply:
Added in 7a9a3595141c5df6ec4eb356cd5f510c02378c78